cypress/unit/store-mutation-initializeColumnToCategoryMap.cy.js

```diff
@@ -0,0 +1,47 @@
+import { mutations } from "~/store/index-refactor";
+
+let state = {};
+
+describe("initializeColumnToCategoryMap", () => {
+
+    beforeEach(() => {
+
+        // Setup
+        state = {
+
+            columnToCategoryMapping: {},
+
+            dataTable: [
+
+                {
+                    "column1Name": "value1", "column2Name": "value2"
+                },
+                {
+                    "column1Name": "value3", "column2Name": "value4"
+                }
+            ]
+        };
+    });
+
+    it("Creates a column to category map based off the data table", () => {
+
+        // Act
+        mutations.initializeColumnToCategoryMap(state, ["column1Name", "column2Name"]);
+
+        // Assert
+        expect(state.columnToCategoryMapping).to.eql({"column1Name": null, "column2Name": null});
```

Comment:
@surchs In this case there is a bit of implicitness going on. Since I'm using `eql` instead of `equal` the comparison will be a deep one.

https://www.chaijs.com/api/bdd/#method_eql

Reply:
Oh wow, thanks for pointing this out. I had no idea.